test/functional/wallet_resendwallettransactions.py

```diff
@@ -33,7 +33,7 @@ def run_test(self):
         time.sleep(1.1)
 
         # Can take a few seconds due to transaction trickling
-        wait_until(lambda: node.p2p.tx_invs_received[txid] >= 1, lock=mininode_lock)
+        self.wait_until(lambda: node.p2p.tx_invs_received[txid] >= 1, lock=mininode_lock)
```

Comment:
Here, `wait_for_broadcast(txid)` would be more appropriate

(txid needs to not be converted to int though)

Reply:
It looks cleaner now! Please have a look.